tools/nxstyle.c

```diff
@@ -135,67 +211,98 @@ int main(int argc, char **argv, char **envp)
   int externc_lineno;   /* Last line where 'extern "C"' declared */
   int linelen;          /* Length of the line */
   int maxline;          /* Lines longer that this generate warnings */
+  bool silent;          /* Used with go not go test option */
   int n;
   int i;
+  int c;
+  extern char *optarg;
+  extern int optopt;
+  extern int optind;
 
-  maxline  = 78;
-  filename = argv[1];
+  g_gonogo    = false;
+  maxline     = 78;
+  filename    = argv[1];
+  silent      = false;
 
```

Comment:
1.remove filename assignment: not valid, and dup with 275
2.change maxline to g_maxline? let's make all command line option to global for consistence
3.Can we merge g_gonogo  and silient into one(e.g. g_verbose)?
g_verbose == 0, no output
g_verbose == 1, PASS/FAIL
g_verbose == 2, each fail line
4.Move the assignment to variable definition:
static int g_verbose = 2;
static int g_maxline = 78;
....

Reply:
Done! ea8b41d8db